src/domain/platform/metrics/utils/getMetricCount.ts

```diff
@@ -5,14 +5,12 @@ export interface Metric {
   value: string;
 }
 
-type MetricTypeName = MetricType[keyof MetricType];
```

Comment:
This type is not a set of the enum keys
![image](https://user-images.githubusercontent.com/16032487/201883540-cebf0577-27ed-474a-bc58-f163548f7780.png)